packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.types.ts

```diff
@@ -119,8 +119,8 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
   onRenderLowerContent?: IRenderFunction<IComboBoxProps>;
 
   /**
-  * Custom width for dropdown. If value is 0, width of the input field is used.
-  * @default 0
+  * Custom width for dropdown. If value is undefined, width of the input field is used.
+  * If value is 0, no width is applied (e.g. the dropdown will be as wide as its content)

```

Comment:
This isn't true anymore since the original logic relied on 0 being a falsey value.

Reply:
What isn't true anymore? Since the previous logic relied on 0 being a falsey value, it made it impossible to tell the dropdown to not specify a size